### PR TITLE
lots of small and a few big things

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Rebalances some of the crashed ships loot area with void reavers 
1. way less free eswords
2. removed a bluecross spawn form the edge of the fallen ship
3. removed oreboxes from the area
4. moved a spotlight aound
5. adds a firstaid kit to the bathroom
6. Adds in some e-shields and hard-spawn voidwolf mods rather then full 13 rng (MEANING 4 hard spawn 8 rng)

image

Fixes a common bug were if you dusted a mob with a AOE type plasma/ion shot that it would trigger on the shooter
Fixes quite a few guns having wrong grips or materials for what they are spirited as or internally have 
Fixes a few gun grips being illogical for the sprite
Removes durasteel refence in the manta ray bluecross gun. As the gun dosnt accually use durasteel

# Added a lot of intercoms and newscasters around the colony map

# Fixes exl implants being on the map
Fixes some maints walls/items
Fixes some misc bad tiles
Connected cutoff maints bits to help with blob/hivemind spawning
Fixed a camera placement and other misc wires/pipes/ect

Allows mecha fab to have better max price reduction: mechs suck and are exspensive

Fixes some printers being able to print things that they shouldnt be able to print

Vastly improved admin test area
1. Adds in way more materals for printing
2. Adds in a church area for church printing stuff
3. Adds in an organ lab for orang printing and splicing

# **Inconstantly upto half reduced all church prints** 
## Sawn down guns are more constant in that sawing down a gun kills its frame and lowers its materials. Sorry Cheaters
_<sub><sup>changes biomatters 50% reduction in printing to a reduction of 25%</sup></sub>_

### CODE REVIEW

**Comment by ChefDoggo** [comment]
-1 for damn sure, if this PR isn't a shitpost than im seriously baffled. removing 2 unique item spawns from scrap and flatout killing the value of going to scraphaven, actively disvalues the i-lathe by giving it to guild and presumably kills the value of a biolathe
this has gotta be a joke PR

**Comment by benj8560** [comment]
hate bioprinter nerf do 70-80% like I suggested. alternatively at LEAST an exemption should be put in church lore allowing factorials to use I-lathes when a bioprinter is available.

Don't remove THAT many of the voidwolf gun spawns cos their RNG based and often end up with 80% scopes on a unlucky day (the scopes are useless nerfed into the ground) you've also made it 10x harder to fight them with the funny ore boxes gone I'd see how prospecting fares getting in there before tweaking the loot.

he cost to produce some church weapons via manifestation needs to be adjusted to their true material costs.

Add the discussion needed title lets be real.

**Comment by FoxBopGaming** [comment]
I'm assuming this: "removes biomatters 50% reduction in printing"
Is in reference to the bioprinter and how it allows a person to print things for cheaper, in which case, getting rid of it sort of invalidates the whole point of the bioprinter, seeing as it's a decent lathe, better then a standard lathe, but far outclassed by higher tier upgraded lathes due to the bioprinter's own inability to be upgraded, if i'm wrong, please let me know.

I get removing ore boxes because of the cheese, but that's a band aid fix and honest makes no sense because it is a MINING POST and therefor should have mining equipment, possibly relocating the oreboxes to a spot more out of the way would work better?

what the hell does this even mean? "Inconstantly upto half reduced all church prints"

Giving I-Lathe creation ability to guild is a touchy subject due to the fact it directly nerfs SI with no recourse or reason, this is assuming it's a base crafting recipie. A possible better way of going about this in order to allow some level of production while SI doesn't exist is by making it disk locked? perhaps being locked away much like the guild's bullet fab is, meaning it could be a crime to get it without GM. Alternatively possibly giving them a disk with limited prints on it, maybe 2-3? or so in order to allow SI to still have some foothold majority over the I-Lathe market and maintain some level of scarcity.

**Comment by benj8560** [comment]
> I'm assuming this: "removes biomatters 50% reduction in printing" Is in reference to the bioprinter and how it allows a person to print things for cheaper, in which case, getting rid of it sort of invalidates the whole point of the bioprinter, seeing as it's a decent lathe, better then a standard lathe, but far outclassed by higher tier upgraded lathes due to the bioprinter's own inability to be upgraded, if i'm wrong, please let me know.
> 
> I get removing ore boxes because of the cheese, but that's a band aid fix and honest makes no sense because it is a MINING POST and therefor should have mining equipment, possibly relocating the oreboxes to a spot more out of the way would work better?
> 
> what the hell does this even mean? "Inconstantly upto half reduced all church prints"
> 
> Giving I-Lathe creation ability to guild is a touchy subject due to the fact it directly nerfs SI with no recourse or reason, this is assuming it's a base crafting recipie. A possible better way of going about this in order to allow some level of production while SI doesn't exist is by making it disk locked? perhaps being locked away much like the guild's bullet fab is, meaning it could be a crime to get it without GM. Alternatively possibly giving them a disk with limited prints on it, maybe 2-3? or so in order to allow SI to still have some foothold majority over the I-Lathe market and maintain some level of scarcity.

The disk infact used to be in the GMs office and at the time was a crime. Everyone hated it and everyone would just hack into the GMs office and grab the disk. No one cares about that. Infact! No one cares about the bullet fab. Colony wants bullets to badly marshals included to give a shit. Everyone just quietly looks the other way even the GMs. Boils down to the same problem longarm laws had can't have a law thats so inconsistently enforced if enforced at all

**Comment by benj8560** [comment]
giving I-lathes to guild is kinda a nice thing over all up until the moment SI exists and needs materials from lonestar. All that guild wants it plasteel and plasteel is cheap and easy to get a shifts worth for them.  Biggest problem is if we nerf (bad idea) Bioprinters there needs to be a accessible alternative

**Comment by Trilbyspaceclone** [comment]
> I'm assuming this: "removes biomatters 50% reduction in printing" Is in reference to the bioprinter and how it allows a person to print things for cheaper, in which case, getting rid of it sort of invalidates the whole point of the bioprinter, seeing as it's a decent lathe, better then a standard lathe, but far outclassed by higher tier upgraded lathes due to the bioprinter's own inability to be upgraded, if i'm wrong, please let me know.
> 

Yes that is correct its refering to the 50% reduction
T3 I-lathen dosnt match it, you need t4 + t3 parts in the I-lathen to match it current.
Meaning you need eather to scav some GP parts/ work with guild or with mining

Note that the whole point of the bioprinter is printing church gear and biomatter like items (excluding organs)


> I get removing ore boxes because of the cheese, but that's a band aid fix and honest makes no sense because it is a MINING POST and therefor should have mining equipment, possibly relocating the oreboxes to a spot more out of the way would work better?
> 

They get ore satchles, and the smelter unit.
Note thats not even meant to be a mining base when I made it, someone else turned it into it for some reason.

> what the hell does this even mean? "Inconstantly upto half reduced all church prints"
> 

This means that prices for church gear is *not constantly* upto half reduced in costs, do to the bioprinter nerf


> Giving I-Lathe creation ability to guild is a touchy subject due to the fact it directly nerfs SI with no recourse or reason, this is assuming it's a base crafting recipie. A possible better way of going about this in order to allow some level of production while SI doesn't exist is by making it disk locked? perhaps being locked away much like the guild's bullet fab is, meaning it could be a crime to get it without GM. Alternatively possibly giving them a disk with limited prints on it, maybe 2-3? or so in order to allow SI to still have some foothold majority over the I-Lathe market and maintain some level of scarcity.
>

Guilds crafting station is open to everyone in the guild
Guilds bullet fab is "A bandaid fix well we have low pop"
I truely do regret giving I-lathens to SI as it wasnt meant to be their mainproduct that everyone buys (when theirs no church)

**Comment by FoxBopGaming** [comment]
> > I'm assuming this: "removes biomatters 50% reduction in printing" Is in reference to the bioprinter and how it allows a person to print things for cheaper, in which case, getting rid of it sort of invalidates the whole point of the bioprinter, seeing as it's a decent lathe, better then a standard lathe, but far outclassed by higher tier upgraded lathes due to the bioprinter's own inability to be upgraded, if i'm wrong, please let me know.
> 
> Yes that is correct its refering to the 50% reduction T3 I-lathen dosnt match it, you need t4 + t3 parts in the I-lathen to match it current. Meaning you need eather to scav some GP parts/ work with guild or with mining

Well this nerf would still make it on par with a normal baseline lathe no? It should still be better than a normal lathe shouldn't it, considering that's still it's thing? unless it's better than a baseline lathe AND has an additional reduction buff on top of the reduction it already had?

**Comment by Trilbyspaceclone** [comment]
Its currently faster and has more que spaces over a normal

**Comment by benj8560** [comment]
for reference a T3 I-lathe sits at 60% mat costs. A T4 gets you 40%. bioprinters sit right in the middle at 50%. I still personally suggest just going to 70% for bioprinters. AKA what i assume a T2 I-lathe would be. I'm not all to bothered about how wonky the print costs would be for biomatter stuff its whatever.

**Comment by benj8560** [comment]
T2 since originally old pre-nerf autolathes would get down to 50% at T2 parts (I think). The original idea behind the buff way back.

**Comment by benj8560** [comment]
it also boils down to soj finally just finally seeing some more pop church players being a sizeable chunk of that. Does soj really need a anti-fun PR rn?

**Comment by Fawkes541** [comment]
- way less free eswords
- removed a bluecross spawn
- removed some armors
- removed oreboxes from the area
- Adds in some e-shields and hard-spawn voidwolf mods rather then full 12 rng
please keep the loot the same while you remove the cheese. cheese removal is fine, but let's remove the cheese first then give it a bit to see how it is. I especially dislike the reduction of the gun mods, this place is one of the only places to voidwolf gun mods, specifically the gauss barrels which are almost required on any competitive gun, if cargo is not on shift. one of the only reasons prospies come here is for the gun mods. the grenades never get used, the suits do get used but that's a personal choice since they slow you down and others are usually availlable (preppers), and the bluecross are just neat. 

if you are dead set on reducing the gun mods so much please take away the forged barrel from the C20 SMGs the wolves drop and add the gauss

you may also have forgotten some ore boxes in the scrap haven ship proper

-     moved a spotlight aound
cool and good
-     adds a firstaid kit to the bathroom
make it a combat or something maybe, balance with how much loot you decide to leave. could add even more meds

I don't play church or guild so I have no comments beyond it's fair that church is getting nerfed a little since they get a maxed lathe for near free at the start of the game, it seems like. 50% is bonkers. I agree with other people who say maybe tweak it to 80 or 75 or somewhere in there, but you could also full remove it to give church a reason to go buy or loot stock parts. 

as for the sawnoff gunframe thing is it possible to keep the frame and destroy the barrel, and maybe the mechanism instead? the entire frame gone is a lil wack.

**Comment by Trilbyspaceclone** [comment]
Combat kit is has tons of chemicals
its to help stop anyone bleeding or need a small patch after clearing it.

When you saw a gun down your sawing the frame down and barrel
Its just keeping it constant with other sawing such as bolts, STS and skrellie

**Comment by Trilbyspaceclone** [comment]
Image in the pr shows most of the direct changes
I did state the movement of the spot light.....
Guass barrels are some of the stronger end spawns so 1 hard isnt that bad

**Comment by SingingSpock** [comment]
Please, for the love of the Absolute, atomize this. This is a _lot_ of changes, some of which are bugfixes and some of which are going to piss off half the server. Let the changes be discussed individually rather than in one mega-PR

**Comment by Trilbyspaceclone** [comment]
> Please, for the love of the Absolute, atomize this. This is a _lot_ of changes, some of which are bugfixes and some of which are going to piss off half the server. Let the changes be discussed individually rather than in one mega-PR

This was atomized
and then it got scope creeped

**Comment by SingingSpock** [comment]
> > Please, for the love of the Absolute, atomize this. This is a _lot_ of changes, some of which are bugfixes and some of which are going to piss off half the server. Let the changes be discussed individually rather than in one mega-PR
> 
> This was atomized and then it got scope creeped

Then un scope creep it and make it as more than one PR, **please**. Mixing important bugfixes with major balance changes that nobody seems to agree with is bad.

**Comment by Fawkes541** [comment]
trilby, I think, correct me if I'm wrong, I could be, you don't really play prospie or outsider, and that makes me think you don't understand how much damage the reavers do, they will full auto your ass with guns with very decent AP, and a standrd medkit is kind of like offering a naked man socks in an alaskan winter. maybe a coat would be a better analogy, or maybe like, fuck, a brisk fall day, but he still is wearing no pants and is cold.  

the real loot here is, in order
- the gauss coils and triggers, the scopes are a joke
- the armor
- the smelter board

everything else here is just gravy. no one actually uses the grenades because it just goes into a prospie armory, the ammo is nice but there are other ways to get that except for the AP 9mm stuff, which no one actually uses from what I've seen, since all the guns people seem to use are rifle cal unless you run across some bonkers handgun like the Rafale or a bluecross. 

in it's current state, scrap haven is a stepping stone that people do to get gear to do harder dungeons. it doesn't let you instantly clear the gauntlet with no problems but it does upgrade the guns that your prospie team already has by giving your team lots of gun mods, plus, you have a chance to find rare guns. I have not heard of anyone who is unhappy with it.  as it is right now, this PR makes it so that the mods here will coil one single person's gun instead of the team's or to sell other coils. all of these mods aren't even that rare but are locked behind lonestar cargo for like, a hundred and fifty creds and favor with a tier one station that most techs unlock in the first few minutes.  if people can go to cargo and blow their starting cash to buy ~3 of these assuming there's a tech awake why should they not also be able to choose to do a dangerous ruin to get the same items without having to rely on a person taking a job that most of the time is not filled.

**Comment by SkinkHunt** [comment]
What exactly is being rebalanced here? This is all stuff the prospies can sell or use. Not only are you reducing all of that, you're not placing alternatives that could at least make up the difference.

**Comment by Trilbyspaceclone** [comment]
> * the gauss coils and triggers, the scopes are a joke

????
Its a joke???

**Comment by Trilbyspaceclone** [comment]
> What exactly is being rebalanced here? This is all stuff the prospies can sell or use. Not only are you reducing all of that, you're not placing alternatives that could at least make up the difference.

The Pr is clearly labled, even put a image with lines and boxes showing the changes

**Comment by Trilbyspaceclone** [comment]
> Then un scope creep it and make it as more than one PR, **please**. Mixing important bugfixes with major balance changes that nobody seems to agree with is bad.


Bug fixs *are* balance changes

**Comment by SkinkHunt** [comment]
> The Pr is clearly labled, even put a image with lines and boxes showing the changes

"Clearly labeled" is not "what is being rebalanced." 

What is being balanced. What aspects of gameplay. Your tags don't answer that.

**Comment by ChefDoggo** [comment]
im basically just seeing a PR thats "i found some bug fixes that should be done but heres a bunch of unwanted salt ontop of it" which is mostly based on info gotten from just watching players instead of actually clearing it yourself, im super against the scrap changes almost strictly because of the fact you lack the actual experience of the gameplay loop. i could see someone make an easier argument if they had the time in prospectors to then effectively "balance" something to make it less free or the like but out of everything scraphaven is just not the place to touch, its a roundstart relatively "_easy_" location to get decent enough roundstart gear to let the team then move into other locations without getting sweeped

this is almost entirely based on "i saw X and Y so im gonna nerf it" its not "i played and experienced X and Y and i think it needs Z" which is in my opinion a horrible way to try and "balance"

**Comment by Fawkes541** [comment]
> > * the gauss coils and triggers, the scopes are a joke
> 
> ???? Its a joke???

there are 3 scopes attatchies in the game, and this scope, while it's the best one for actual zoom, nerfs your fire rate by 40%, which to most people is not worth it for your standard dungeon clearing AR unless they're trying to make a funny gun. putting this mod on a gun is turning it into a DMR that is worse than a DMR since most already have scopes. some guns don't even have scope slots to use this on, while all of the ballistic guns in the game can use and benefit from the trigger and the gauss barrel. this is why it's a joke. it's not worthless loot since you can probably pawn it off to some marshals or blackshield for their funny sniper rifle but it's a niche use while the other ones are very much required on any ballistic you want to perform competitively.

**Comment by Fawkes541** [comment]
again, I'm not arguing against the removal of the ore crates for the cheese, but nerfing the loot AND making a dungeon that people already like as hard as it should be is not the way

**Comment by SkinkHunt** [comment]
The gauss and the triggers make up for the fire rate loss of anything with that scope, that's why I asked what exactly is being balanced here. Very few things benefit from the gauss barrel alone, unless its a Greyson or BlueCross gun its slapped on. Most of the time, prospies use it on their more favored gun to make it more viable.

**Comment by Fawkes541** [comment]
> The gauss and the triggers make up for the fire rate loss of anything with that scope, that's why I asked what exactly is being balanced here. Very few things benefit from the gauss barrel alone, unless its a Greyson or BlueCross gun its slapped on. Most of the time, prospies use it on their more favored gun to make it more viable.

`GUN_UPGRADE_DAMAGE_BRUTE = 5,
		GUN_UPGRADE_PEN_MULT = 1.2,
		GUN_UPGRADE_PIERC_MULT = 1,
		GUN_UPGRADE_FIRE_DELAY_MULT = 1.2,
		GUN_UPGRADE_RECOIL = 1.2,`
this is what the gauss barrel does. 
`GUN_UPGRADE_FIRE_DELAY_MULT = 0.8,
		GUN_UPGRADE_FORCESAFETY = FALSE,
		)`
this is what the trigger does.
the trigger and the gauss balance each other out and buff the gun by a 20% pen increase with an extra 5 damage, the scope is just objectively ass

**Comment by ChefDoggo** [comment]
> > The gauss and the triggers make up for the fire rate loss of anything with that scope, that's why I asked what exactly is being balanced here. Very few things benefit from the gauss barrel alone, unless its a Greyson or BlueCross gun its slapped on. Most of the time, prospies use it on their more favored gun to make it more viable.
> 
> `GUN_UPGRADE_DAMAGE_BRUTE = 5, GUN_UPGRADE_PEN_MULT = 1.2, GUN_UPGRADE_PIERC_MULT = 1, GUN_UPGRADE_FIRE_DELAY_MULT = 1.2, GUN_UPGRADE_RECOIL = 1.2,` this is what the gauss barrel does. `GUN_UPGRADE_FIRE_DELAY_MULT = 0.8, GUN_UPGRADE_FORCESAFETY = FALSE, )` this is what the trigger does. the trigger and the gauss balance each other out and buff the gun by a 20% pen increase with an extra 5 damage, the scope is just objectively ass

its good basically only on bolt action rifles where firing rate is afaik unaffected, besides that yeah the scope is basically the worst pick

**Comment by Trilbyspaceclone** [comment]
> > The Pr is clearly labled, even put a image with lines and boxes showing the changes
> 
> "Clearly labeled" is not "what is being rebalanced."
> 
> What is being balanced. What aspects of gameplay. Your tags don't answer that.

Litterly labled every change with a photo as well.
Its clearly labled if your having issues reading then i cant help you further

**Comment by SingingSpock** [comment]
> > Then un scope creep it and make it as more than one PR, **please**. Mixing important bugfixes with major balance changes that nobody seems to agree with is bad.
> 
> Bug fixs _are_ balance changes

Fixing plasma shots and improving the admin area are not even in the same general category as rebalancing a dungeon's loot, doubling the price of all goods produced with a bioprinter, and improving the material efficiency of mech fabs. This is **at least** 4-5 PRs worth of changes, and it should not be done in one mega-PR.

**Comment by Trilbyspaceclone** [comment]
> > > Then un scope creep it and make it as more than one PR, **please**. Mixing important bugfixes with major balance changes that nobody seems to agree with is bad.
> > 
> > 
> > Bug fixs _are_ balance changes
> 
> Fixing plasma shots and improving the admin area are not even in the same general category as rebalancing a dungeon's loot, doubling the price of all goods produced with a bioprinter, and improving the material efficiency of mech fabs. This is **at least** 4-5 PRs worth of changes, and it should not be done in one mega-PR.

Yes they should.

**Comment by SkinkHunt** [comment]
> > > > Then un scope creep it and make it as more than one PR, **please**. Mixing important bugfixes with major balance changes that nobody seems to agree with is bad.
> > > 
> > > 
> > > Bug fixs _are_ balance changes
> > 
> > 
> > Fixing plasma shots and improving the admin area are not even in the same general category as rebalancing a dungeon's loot, doubling the price of all goods produced with a bioprinter, and improving the material efficiency of mech fabs. This is **at least** 4-5 PRs worth of changes, and it should not be done in one mega-PR.
> 
> Yes they should.

No, no they shouldn't.

**Comment by SkinkHunt** [comment]
> > > The Pr is clearly labled, even put a image with lines and boxes showing the changes
> > 
> > 
> > "Clearly labeled" is not "what is being rebalanced."
> > What is being balanced. What aspects of gameplay. Your tags don't answer that.
> 
> Litterly labled every change with a photo as well. Its clearly labled if your having issues reading then i cant help you further

Yes. I see what you are removing. Why, is still a mystery. Why remove the gauss barrels? How is removing them a "balance"? 
EG >removing gauss because [gameplay related reason]. I am not in your brain, Trilby.  I am not psychic. If this is the struggle I get with a PR multiple people complain to me about, I'll just elect to reject it no matter whats in it.

**Comment by Trilbyspaceclone** [comment]
> Why remove the gauss barrels

Please read the pr again.

**Comment by SkinkHunt** [comment]
> > Why remove the gauss barrels
> 
> Please read the pr again.

Lets not play this circular logic game. "Balance" is not a valid answer, it is undefined, since you have yet to define with gameplay rational.

**Comment by cdb-is-not-good** [comment]
There's a lotta stuff about this PR that I'm not huge on. I don't like making i-lathes more common again without nerfing their efficiency, I don't really....see why we're nerfing the biolathe, efficiency and on the back of that change if the idea is to make it less efficient but lower costs of church stuff why....is it not consistent? I'm seeing cost changes from 60-50% and not for every ingredient?

I'm kinda lowkey fine with the mech-fab efficiency nerf.

I'm _not_ super keen on making the I-lathe more common as stated before unless we're going to take a serious look at making recycling less insanely good on an upgraded lathe.

4 e-cutlasses to 1 e-cutlass might be fine *if* you unnerfed them. They were nerfed in light of how common they were but that was b efore we made them significantly rarer.

3 voidwolf suits to 2  is...fine, tbh. Minor change.

Getting rid of the bluecross is...eh....It's a pretty tough fight even with cheese due to how dnagerous the mobs are. 

I'm fine with getting rid of the ore boxes for cheese but wouldn't it make more sense to simply make it so that the ore-box is not an invincible piece of cover? Perhaps make it so that it can be penetrated by bullets or has a pool of health and isnt invulnerable.

12 random voidwolf mods to 3 seems a bit harsh, but there ARE so many goddamn mods in that pile it's pretty ridiculous so i'm less bothered by this in light of some hardspawns.

the sawnoff changes seem fine

File: code/game/machinery/bioprinter_nt.dm [hunk 1]
```diff
@@ -4,11 +4,11 @@
 	icon_state = "bioprinter"
 	unsuitable_materials = list()
 	build_type = AUTOLATHE | BIOPRINTER
+	code_dex = "MECHFAB"
 	storage_capacity = 480
 	speed = 11
 	have_recycling = TRUE
 	queue_max = 16 //Might be 8 in game do to wires
```
Comment (cdb-is-not-good) [inline code/game/machinery/bioprinter_nt.dm:11]:
```suggestion
	queue_max = 16 //Might be 8 in game do to wires
	mat_efficiency = 0.75
```

**Comment by SingingSpock** [comment]
This is still an omni-PR and it really needs to not be.

**Comment by Vex8tion** [comment]
> This is still an omni-PR and it really needs to not be.

Gonna second this and toss in my two cents as well. I don't have anything to say as far as loot spawns go, as I've never played prospector, but the scope for this PR is impossible to ascertain, especially when no actual rationale has been provided for any of the changes. Lumping a dozen different, varied, unrelated changes into one PR turns that PR into a nightmare for review and actual discussion, as can be pretty well seen here. This should've been done piecemeal in a number of smaller PRs.

The Church changes are the only things I can really speak on with any real experience, and if I'm honest, it... makes no sense the way it's been done. What is the purpose to nerfing bioprinter efficiency... then lowering the material cost values for Church-printed items? What's worse is that the rebalance of item costs seems basically random. Is there a reason not to rebalance costs uniformly?

**Comment by Trilbyspaceclone** [comment]
> The Church changes are the only things I can really speak on with any real experience, and if I'm honest, it... makes no sense the way it's been done. What is the purpose to nerfing bioprinter efficiency... then lowering the material cost values for Church-printed items? What's worse is that the rebalance of item costs seems basically random. Is there a reason not to rebalance costs uniformly?


Its so it doesn't as heavily affect church items, as they were all with few exceptions balanced around the 50% reduction
So by trying remove the added assumed material cost it makes the church not doble nerfed to have an in affect difference of 150% costs to print

Some stuff was lowered *more then 50%* based on judgment of the gear (Cells, some armors, some guns) some was less such as already cheap guns

Church bioprinter for some silver, metal, glass and a toolbox is better thena  t3 I-lathen round start located entirely in 1 department 

"Inconstantly upto half reduced all church prints"
is right in the pr desc, in the largest size

**Comment by Vex8tion** [comment]
> > The Church changes are the only things I can really speak on with any real experience, and if I'm honest, it... makes no sense the way it's been done. What is the purpose to nerfing bioprinter efficiency... then lowering the material cost values for Church-printed items? What's worse is that the rebalance of item costs seems basically random. Is there a reason not to rebalance costs uniformly?
> 
> Its so it doesn't as heavily affect church items, as they were all with few exceptions balanced around the 50% reduction So by trying remove the added assumed material cost it makes the church not doble nerfed to have an in affect difference of 150% costs to print
> 
> Some stuff was lowered _more then 50%_ based on judgment of the gear (Cells, some armors, some guns) some was less such as already cheap guns

Thank you for the explanation. I'm not sure I entirely agree with the reasoning (it seems like an unnecessary amount of effort to go in and manually tweak _every single Church item_ going forward) but I appreciate the explanation nonetheless.

> > Church bioprinter for some silver, metal, glass and a toolbox is better thena t3 I-lathen round start located entirely in 1 department

I'll point out that the Church is in no way restricted from providing these bioprinters to the other departments as they see fit. It's hardly something they hoard, either. I'll freely admit I'm not 100% up to date on the importance of i-lathe tiers, but if the issue is that it's just in one department... people outside the Church make use of their bioprinter via orders all the time. Additionally, as many people have pointed out, the Church printer cannot be upgraded.

> > "Inconstantly upto half reduced all church prints" is right in the pr desc, in the largest size
I'll ask you respectfully not to be petty in this way. Ask for clarification instead of being snarky in a response. One line is hardly an explanation or rationale, it's only a statement about what's being done, and for wide-ranging balance changes like these, it's generally a good idea to have a little more up-front than that. It helps avoid a lot of the back-and-forth in this very PR's conversations to have a clearly stated reason for doing things.

**Comment by Trilbyspaceclone** [comment]
>I'll point out that the Church is in no way restricted from providing these bioprinters to the other departments as they see fit. It's hardly something they hoard, either. I'll freely admit I'm not 100% up to date on the importance of i-lathe tiers, but if the issue is that it's just in one department... people outside the Church make use of their bioprinter via orders all the time. Additionally, as many people have pointed out, the Church printer cannot be upgraded.
>

Well thats not an issue! Far from it, i mind it being unfeatured and easy to produce, if its power level wasnt on par with a t4 I-auto something that takes multilabel departments to work with or at the lest should if you ignore solo SI scaving GP parts

>I'll ask you respectfully not to be petty in this way.
>
Im not being petty, im pointing out that its labled, and not a hidden buff/nerf
"What's worse is that the rebalance of item costs seems basically random. Is there a reason not to rebalance costs uniformly?"
was answered with `thy jugement` more or less, im just clarifing that its not *meant* to be uniformly as some changes are done under different case by case I.E Cells getting a 90%ish reduction in plasma well the other values stay the same
Husslar got reduced to 100% (tho now its more like 75%) with the 25% added effectness making it *in affect* 3.5t

File: code/modules/research/rdmachines.dm [hunk 2]
```diff
@@ -42,17 +42,19 @@
...
 
+	code_dex = "PROTOLATHE"
+
 
 /obj/machinery/autolathe/rnd/imprinter
 	name = "circuit imprinter"
 	desc = "A machine used for printing advanced circuit boards. Operated from an R\&D console."
 	icon_state = "imprinter"
 	circuit = /obj/item/circuitboard/circuit_imprinter
 	max_efficiency = 0.2
```
Comment (Trilbyspaceclone) [inline code/modules/research/rdmachines.dm:53]:
```suggestion
	max_efficiency = 0.2
	build_type = IMPRINTER
```

**Comment by Fawkes541** [comment]
raided haven as a prospie on the TM. gear was standard armor + nanogate armor + nanogate regen + recoil modded Bond caseless rifle. cleared the junk heap and up to the first room with scope cheese + bond is just generally alright. took some damage from one of the full auto ones in the last room who was in the back, would have died if I did not have QC and the nanogate regen. made a 2x2 of a plasteel wall and 3 steel walls from the scrap to use as cover, was not quick enough to kill the wolves before they shot through one layer, leaving one layer, which they can pen. killed 4 of the voidwolves before I died to organ damage and blood loss. 

would be harder than it was but not impossible to clear with two people drawing aggro. in a fine state if you leave the loot the same or buff it for the extra trouble, until we find the meta with it. building our own cover sucks a little, maybe put a large pillar in the room.

**Comment by benj8560** [comment]
wanted to give this a **now** plus 1. compromises were made in most of the problems foke had I don't see any problems with it aside from maybe touching the loot at Reavers (still not a huge deal) everything else seems fine to me. There's many important fixes such as preventing bioprinters from doing organs.

 A nerf to bioprinters was long overdue it was likely just never done due to all the work in rebalancing the biomatter print costs since those numbers as stated where balanced around 50% costs.  All autolathes got this treatment awhile back except for these.

**Pull request closed without merging**